lib/verify.js

```diff
@@ -30,6 +30,8 @@ function verify_metadata(metadata) {
         assert(!!metadata['document'], "metadata missing document");
     } else if (object_type === 'ImageObject') {
         assert(!!metadata['cdnFilename'], "Image object missing cdnFilename (has no data?)");
+    } else if (object_type === "DictionaryObject") {
+        assert(!!metadata['word'], "metadata missing word");
```

Comment:
Good point, I'm going to leave off "part of speech" as that doesn't seem as important as a word/wordDefinition pair.